fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -429,95 +731,228 @@ public class OperationType {
+
+    @IgnorableOnReplayFailed
     public static final short OP_DROP_BROKER_V2 = 13032;
+
+    @IgnorableOnReplayFailed
     public static final short OP_DECOMMISSION_DISK = 13033;
+
+    @IgnorableOnReplayFailed
     public static final short OP_CANCEL_DECOMMISSION_DISK = 13034;
+
+    @IgnorableOnReplayFailed
     public static final short OP_DISABLE_DISK = 13035;
+
+    @IgnorableOnReplayFailed
     public static final short OP_CANCEL_DISABLE_DISK = 13036;
-    public static final short OP_BATCH_DELETE_REPLICA_BATCH = 13037;
 
+    public static final short OP_BATCH_DELETE_REPLICA = 13037;
+
+    @IgnorableOnReplayFailed
     public static final short OP_TIMESTAMP_V2 = 13040;
+
     public static final short OP_LEADER_INFO_CHANGE_V2 = 13041;
 
+    @IgnorableOnReplayFailed
     public static final short OP_ADD_FUNCTION_V2 = 13050;
+
+    @IgnorableOnReplayFailed
     public static final short OP_DROP_FUNCTION_V2 = 13051;
 
     public static final short OP_DROP_USER_V3 = 13060;
+
+    @IgnorableOnReplayFailed
     public static final short OP_UPDATE_USER_PROP_V3 = 13061;
 
+    @IgnorableOnReplayFailed
     public static final short OP_CREATE_STREAM_LOAD_TASK_V2 = 13070;
 
     // storage volume
+
+    @IgnorableOnReplayFailed
     public static final short OP_SET_DEFAULT_STORAGE_VOLUME = 13100;
+
+    @IgnorableOnReplayFailed
     public static final short OP_CREATE_STORAGE_VOLUME = 13101;
+
+    @IgnorableOnReplayFailed
     public static final short OP_UPDATE_STORAGE_VOLUME = 13102;
+
+    @IgnorableOnReplayFailed
     public static final short OP_DROP_STORAGE_VOLUME = 13103;
 
     // Pipe operations log
+    @IgnorableOnReplayFailed
     public static final short OP_PIPE = 12200;
 
     // Primary key
+    @IgnorableOnReplayFailed
     public static final short OP_MODIFY_PRIMARY_INDEX_CACHE_EXPIRE_SEC = 13200;
 
     // alter catalog
+    @IgnorableOnReplayFailed
     public static final short OP_ALTER_CATALOG = 13300;
 
     // Dictionary
+
+    @IgnorableOnReplayFailed
     public static final short OP_CREATE_DICTIONARY = 13400;
+
+    @IgnorableOnReplayFailed
     public static final short OP_DROP_DICTIONARY = 13401;
+
+    @IgnorableOnReplayFailed
     public static final short OP_MODIFY_DICTIONARY_MGR = 13402;
 
     // Replication job
+    @IgnorableOnReplayFailed
     public static final short OP_REPLICATION_JOB = 13500;
 
     /**
      * NOTICE: OperationType cannot use a value exceeding 20000, and an error will be reported if it exceeds
      */
     public static final short OP_TYPE_EOF = 20000;
+
+    public static final ImmutableSet<Short> IGNORABLE_OPERATIONS = buildIgnorableOperations();
+
+    private static ImmutableSet<Short> buildIgnorableOperations() {
+        ImmutableSet.Builder<Short> builder = ImmutableSet.builder();
+        Set<Short> allOperations = new HashSet<>();
+
+        for (Field field : OperationType.class.getFields()) {
+            if (field.getType() != short.class) {
```

Comment:
if field.getName() not starts with "OP_" , continue?

Reply:
done